root/components/relationship-editor.tt

```diff
@@ -10,11 +10,11 @@
 </div>
 
 <script type="text/html" id="template.relationship-dialog">
-  <!-- ko if: relationship().editsPending -->
+  <!-- ko if: $data.editing && $data.editing.editsPending -->
```

Comment:
This fixes the bug. `$data` is the dialog, `editing` is a reference to the relationship being edited.
https://github.com/mwiencek/musicbrainz-server/blob/mbs-7907/root/static/scripts/relationship-editor/common/dialog.js#L570